### PR TITLE
Ria 3846 organisation policy update

### PR DESCRIPTION
**Pull request by bradsorour**
### JIRA link (if applicable) ###

[RIA-3846](https://tools.hmcts.net/jira/browse/RIA-3846)

### Change description ###

See Jira

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: src/main/java/uk/gov/hmcts/reform/iacaseapi/domain/handlers/presubmit/LegalRepOrganisationFormatter.java [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+    }
+
+    @Override
+    public PreSubmitCallbackResponse<AsylumCase> handle(PreSubmitCallbackStage callbackStage,
+                                                        Callback<AsylumCase> callback) {
+
+        if (!canHandle(callbackStage, callback)) {
+            throw new IllegalStateException("Cannot handle callback");
+        }
+
```
Comment (makowalski) [inline src/main/java/uk/gov/hmcts/reform/iacaseapi/domain/handlers/presubmit/LegalRepOrganisationFormatter.java:46]: @bradsorour can you add OrganisationPolicy set up to this handler? I think that way we can call PRD endpoint (`retrieve()` method) one time and use it for two goals.

Comment (bradsorour) [reply src/main/java/uk/gov/hmcts/reform/iacaseapi/domain/handlers/presubmit/LegalRepOrganisationFormatter.java:48]: Yes, indeed. I've moved OrganisationPolicy into this handler.

**Review by makowalski — DISMISSED** [review]
LGTM

**Review by mkayad — DISMISSED** [review]
LGTM

**Pull request closed without merging**